tests/e2e/specs/upe-split/shopper/shopper-deferred-intent-creation-upe-enabled.spec.js

```diff
@@ -18,13 +18,10 @@ import { uiUnblocked } from '@woocommerce/e2e-utils/build/page-utils';
 const { shopper, merchant } = require( '@woocommerce/e2e-utils' );
 
 const UPE_METHOD_CHECKBOXES = [
-	'#inspector-checkbox-control-5', // bancontact
-	'#inspector-checkbox-control-6', // eps
 	'#inspector-checkbox-control-7', // giropay
-	'#inspector-checkbox-control-8', // ideal
-	'#inspector-checkbox-control-9', // sofort
```

Comment:
I removed four UPE methods because 
* they are not used in the tests and just taking time needed to enablement/disablement
* they should be ideally added/integrated into the test suite once corresponding areas for particular payment methods are being added https://github.com/Automattic/woocommerce-payments/issues/6727